lib/fennec/stun/auth.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Fennec.STUN.Auth do
+  @moduledoc """
+  This module implements authentication and authorization of the STUN protocol.
+  """
+
+  alias Jerboa.Format.Body.Attribute
+  alias Jerboa.Format.Body.Attribute.{Username, Nonce, Realm}
+  alias Jerboa.Params
+
+  def get_secret do
+    Application.get_env(:fennec, :secret)
+  end
+
+  def authorize(params, _turn_state) do
+    with u = %Username{} <- Params.get_attr(params, Username),
+         r = %Realm{}    <- Params.get_attr(params, Realm) do
+      # Right now, any authenticated user will do
+      {:ok, %Params{params | attributes: params.attributes -- [u, r]}}
```

Comment:
To simplify request processing. All `TURN` requests have to fail if there is unknown attribute, therefore all request would need to process optional `Username` and `Realm`. Since currently there is no logic in request handlers that make use of those attributes, it's much simpler to just discard them just after authorisation. If we ever would need to know which user is making the request, we could just change it then.